src/components/frame/v5/pages/UserCryptoToFiatPage/partials/BankDetailsModal/useBankDetailsFields.tsx

```diff
@@ -161,6 +164,8 @@ export const useBankDetailsFields = ({
         />,
       );
 
+      refetchKycData();
```

Comment:
Is this necessary given this query should already be refetched as it's specified `refetchQueries` option?

Reply:
Oh I didn't notice that you added that bit in your PR when I was rebasing. I shall remove it, thanks for the nudge 😄